Sources/Display.php

```diff
@@ -906,6 +906,11 @@ function Display()
 			$DBascending = $ascending;
 			$page_operator = $ascending ? '>=' : '<=';
 		}
+		elseif ($start_char === 'L')
+		{
+			$DBascending = !$ascending;
+			$page_operator = $ascending ? '>=' : '<=';
+		}
```

Comment:
Since the only possible values for `$start_char` are `M`, `C`, and `L`, the `else` statement after this line will never be reached. This means that you can replace this entire `if ... elseif ... else` block with two lines:
```
$DBascending = ($start_char === 'M' || $start_char === 'C') ? $ascending : !$ascending;
$page_operator = $ascending ? '>=' : '<=';
```